src/txn/txn_rollback_to_stable.c

```diff
@@ -559,14 +571,15 @@ __rollback_abort_newer_row_leaf(
      * since the page was read from disk.
      */
     WT_ROW_FOREACH (page, rip, i) {
+        stable_update_found = false;
```

Comment:
Please initialize in the called function.

Reply:
Fixed.